apps/ssr-test-app/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": false,
```

Comment:
>false [](http://example.com/codeflow?start=14&length=5)

Is there a good reason to not use strict rules for this new app?

Reply:
Nope; strict enabled!